bmap-parser/README.md

```diff
@@ -0,0 +1,18 @@
+# bmap-parser
+
+The bmap-parser crate aims to implement tools related to bmap. The inspiration for it
```

Comment:
bmap-parser doesn't implement tools, it implements the parsing bits for bmap files :-)